providers/core.tcpsocket.js

```diff
@@ -262,12 +286,25 @@ Socket_chrome.errorStringOfCode = function(code) {
       'UNKNOWN';
 };
 
+/**
+ * Get the Freedom error code associated with a Chrome error string
+ * @method freedomErrorCode
+ * @static
+ * @private
+ * @param {string} errorString The Chrome error string
```

Comment:
this is presumably `String=` in the same way that it is `Number=` in errorStringOfCode?